pkg/catalog/catalog.go

```diff
@@ -54,7 +57,7 @@ func Exists(client *occlient.Client, componentType string) (bool, error) {
 	}
 
 	for _, supported := range catalogList {
-		if componentType == supported.Name {
+		if componentType == supported.Name || componentType == fmt.Sprintf("%s/%s", supported.Namespace, supported.Name) {
```

Comment:
does this mean that we can supply (for example) using: `odo create openshift/nodejs:8.0.0 foobar` and it'd work?

Reply:
if so, this should be documented somewhere

Reply:
> does this mean that we can supply (for example) using: odo create openshift/nodejs:8.0.0 foobar and it'd work?

Yes, where component name will be `foobar` and component is created using `nodejs` builderimagestream of version `8.0.0` picked up from `openshift` namespace.

> if so, this should be documented somewhere

Do you mean documenting in the command for use during cmd help?
Or you suggest putting it in a asciidoc?(If so, I'll raise a bug to track it so that the PR can be unblocked.. Would it be ok?)

Reply:
@cdrage Done as in https://github.com/redhat-developer/odo/pull/673

Reply:
@anmolbabu Not done as this information needs to be in `cmd/create.go` as well, not just external documentation

Reply:
Ok Done